js/apps/system/_admin/aardvark/APP/frontend/html/body.html

```diff
@@ -1,89 +1,91 @@
-    <nav class="navbar" style="display: none">
-      <div class="primary">
-          <div class="navlogo">
-            <a class="logo big" href="#"><img id="ArangoDBLogo" class="arangodbLogo" src="img/arangodb-edition-optimized.svg"/></a>
-            <a class="logo small" href="#"><img class="arangodbLogo" src="img/arangodb_logo_small.png"/></a>
-            <a class="version"><span id="currentVersion"></span></a>
-          </div>
-          <!-- <div id="progressPlaceholderIcon"></div> -->
-            <div class="statmenu" id="statisticBar">
-            </div>
-            <div class="navmenu" id="navigationBar">
-            </div>
+<nav class="navbar" style="display: none">
+  <div class="primary">
+      <div class="navlogo">
+        <a class="logo big" href="#"><img id="ArangoDBLogo" class="arangodbLogo" src="img/arangodb-edition-optimized.svg"/></a>
+        <a class="logo small" href="#"><img class="arangodbLogo" src="img/arangodb_logo_small.png"/></a>
+        <a class="version"><span id="currentVersion"></span></a>
       </div>
-    </nav>
-
-    <div id="modalPlaceholder">
-    </div>
-
-    <div class="bodyWrapper" style="display: none">
-
-      <div class="centralRow">
-
-        <div id='navbar2' class="navbarWrapper secondary">
-            <div class="subnavmenu" id="subNavigationBar"></div>
+      <!-- <div id="progressPlaceholderIcon"></div> -->
+        <div class="statmenu" id="statisticBar">
+        </div>
+        <div class="navmenu" id="navigationBar">
         </div>
+  </div>
+</nav>
 
-        <div class="resizecontainer contentWrapper">
+<div id="modalPlaceholder">
+</div>
 
-          <div id="loadingScreen" class="loadingScreen" style="display: none">
-            <i class="fa fa-circle-o-notch fa-spin fa-3x fa-fw margin-bottom"></i>
-            <span class="sr-only">Loading...</span>
-          </div>
+<div class="bodyWrapper" style="display: none">
 
-          <div id="content" class="centralContent"></div>
+  <div class="centralRow">
 
-          <footer class="footer">
-            <div id="footerBar"></div>
-          </footer>
+    <div id='navbar2' class="navbarWrapper secondary">
+        <div class="subnavmenu" id="subNavigationBar"></div>
+    </div>
 
-        </div>
+    <div class="resizecontainer contentWrapper">
 
+      <div id="loadingScreen" class="loadingScreen" style="display: none">
+        <i class="fa fa-circle-o-notch fa-spin fa-3x fa-fw margin-bottom"></i>
+        <span class="sr-only">Loading...</span>
       </div>
 
+      <div id="content" class="centralContent"></div>
+
+      <script src="https://gw.alipayobjects.com/os/lib/antv/g6/4.3.5/dist/g6.min.js"></script>
```

Comment:
We should not include remote libraries. (e.g. think about a restricted network only arangodb instance).